### ISSUE TITLE
Duplicate export is classed as an error

### ISSUE THREAD

**Issue by garyb** [Pauan/rollup-plugin-purs#14]
Continued from https://github.com/Pauan/rollup-plugin-purs/issues/6#issuecomment-294307356

I didn't open an issue for this before as I wasn't sure if it's something we should be lax about, but the specific error is:

```
�   (purs plugin) Variable b64tohex is already exported
node_modules\jsrsasign\lib\jsrsasign.js
```

If you think we should just submit a fix in the library and not do something in the plugin I understand, feel free to close or whatever!

**Comment by Pauan** [comment]
Yeah, I don't like being lax either, but the thing about `rollup-plugin-purs` is that it converts CommonJS to ES6, and in CommonJS the following is completely valid:

```js
exports.foo = 1;
exports.foo = 2;
```

That will be converted into this:

```js
export var foo = 1;
export var foo = 2;
```

This then triggers the duplicate export error.

However, it could convert it into this instead:

```js
export var foo = 1;
foo = 2;
```

This is reasonable, even by ES6 standards, and`rollup-plugin-purs` already supports weird CommonJS patterns which don't perfectly fit with ES6 modules (such as exporting a variable when *not* at the top level), so I'm okay with it.

**Comment by Pauan** [comment]
This has been fixed by version `1.0.38`

**Comment by garyb** [comment]
Thanks!
